tools/README.md

```diff
@@ -1,7 +1,19 @@
 # Discovery Tools
 
+This is a Go interface implementation to resolve image reference name.
+
+## Build
+
+Developer can build up the binary tools via source code.
+The build should be done under project root directory.
+```
+$ make oci-discovery
+```
+
 ## Example
 
+The built binary tool provides sub-command `resolve` to resolve the image name and fetch the CAS root objects.
```

Comment:
"`ovi-discovery` provides" is shorter and (at least for me) easier to understand than "The built binary tool provides".

Also "sub-command" -> "the subcommand".

And nothing CAS is happening in resolution.  The Merkle root is just the pointer into CAS, it's not in CAS itself.  So I'd prefer "resolve the image name to potential Merkle roots."